bot/seasons/evergreen/movie.py

```diff
@@ -0,0 +1,202 @@
+import logging
+import random
+from enum import Enum
+from os import environ
+from typing import Any, Dict, List, Tuple
+from urllib.parse import urlencode
+
+from aiohttp import ClientSession
+from discord import Embed
+from discord.ext.commands import Bot, Cog, Context, command
+
+from bot.pagination import ImagePaginator
+
+# Get TMDB API key from .env
+TMDB_API_KEY = environ.get('TMDB_API_KEY')
+
+# Define base URL of TMDB
+BASE_URL = "https://api.themoviedb.org/3/"
+
+logger = logging.getLogger(__name__)
+
+# Define movie params, that will be used for every movie request
+MOVIE_PARAMS = {
+    "api_key": TMDB_API_KEY,
+    "language": "en-US"
+}
+
+
+class MovieGenres(Enum):
+    """Movies Genre names and IDs."""
+
+    Action = "28"
+    Adventure = "12"
+    Animation = "16"
+    Comedy = "35"
+    Crime = "80"
+    Documentary = "99"
+    Drama = "18"
+    Family = "10751"
+    Fantasy = "14"
+    History = "36"
+    Horror = "27"
+    Music = "10402"
+    Mystery = "9648"
+    Romance = "10749"
+    Science = "878"
+    Thriller = "53"
+    Western = "37"
+
+
+class Movie(Cog):
+    """Movie Cog contains movies command that grab random movies from TMDB."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+    @command(name='movies', aliases=['movie'])
+    async def movies(self, ctx: Context, genre: str = "", amount: int = 5) -> None:
+        """
+        Get random movies by specifing genre.
+
+        Also support amount parameter,
+        that define how much movies will be shown. Default 5
+
+        Available genres:
+        - Action
+        - Adventure
+        - Animation
+        - Comedy
+        - Crime
+        - Documentary
+        - Drama
+        - Family
+        - Fantasy
+        - History
+        - Horror
+        - Music
+        - Mystery
+        - Romance
+        - Science
+        - Thriller
+        - Western
+        """
+        # Check is there more than 20 movies specified, due TMDB return 20 movies
+        # per page, so this is max. Also you can't get less movies than 1, just logic
+        if amount > 20:
+            await ctx.send("You can't get more than 20 movies at once. (TMDB limits)")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 movies. Just logic.")
+            return
+
+        # Capitalize genre for getting data from Enum, get random page
+        genre = genre.capitalize()
+        page = random.randint(1, 200)
+
+        # Get movies list from TMDB, check is results key in result. When not, raise error. When genre not exist,
+        # show help.
+        try:
+            movies = await self.get_movies_list(self.http_session, MovieGenres[genre].value, page)
+        except KeyError:
```

Comment:
While this is probably sufficient here, it's probably a better idea in general to examine the [API response code](https://www.themoviedb.org/documentation/api/status-codes).

Perhaps a target for an enhancement PR if it becomes a problem in the future. For now it's worth logging the API response if a problem occurs.